etc/twcmanager/config.json

```diff
@@ -304,14 +317,6 @@
       #   "charge_amps": "settings.nonScheduledAmpsMax",
       #   "charge_limit": "config.nonScheduledLimit" },
 
-      # { "name": "Track Green Energy",
```

Comment:
Frack, now that I take a closer look, I probably shouldn't have removed this.  I should check whether there's a cleaner way to implement this, though.